probatus/utils/sampling.py

```diff
@@ -0,0 +1,102 @@
+"""
+Utility functions for sampling
+
+This module holds utility functions for sampling data. They can be used, for
+example, to provide a random single typical datapoin in a technical report.
+"""
+
+import logging
+
+import pandas as pd
+from pandas.api.types import is_numeric_dtype
+
+
+def sample_row(
+    X: pd.DataFrame,
+    filter_rows_with_na: bool = False,
+    random_state: int = 42,
+    max_field_len: int = 50,
+) -> pd.DataFrame:
+    """Samples a pandas dataframe.
+
+    Extracts the column name, datatype, minimum and maximum values for each
+    column in the supplied dataframe. Can generate a sample for use in
+    technical model documentation.
+
+    Example:
+    
+    ```python
+    iris = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv')
```

Comment:
Not sure if it is a good idea to put link like that. I am afraid it might break if they remove the file.

Reply:
Done, used `sklearn.datasets.load_iris()`